blocks/components/editable/index.js

```diff
@@ -271,14 +262,26 @@ export default class Editable extends wp.element.Component {
 		return !! this.state.formats[ format ];
 	}
 
-	toggleFormat( format ) {
+	changeFormats( formats ) {
 		this.editor.focus();
 
-		if ( this.isFormatActive( format ) ) {
-			this.editor.formatter.remove( format );
-		} else {
-			this.editor.formatter.apply( format );
-		}
+		Object.keys( formats ).forEach( format => {
+			const formatValue = formats[ format ];
+			if ( format === 'link' ) {
+				if ( formatValue !== undefined ) {
```

Comment:
To this and the above `TODO`, what's the use case for an empty link? Should we just check general truthiness here, including treating an empty string as an unlink intent?